PrePaper/2023/curveadm palybook 使用指南.md

```diff
@@ -0,0 +1,403 @@
+# curveadm palybook 使用指南
+
+- [curveadm palybook 使用指南](#curveadm-palybook-使用指南)
+  - [curveadm playbook 介绍](#curveadm-playbook-介绍)
+  - [例子一：使用 curveadm playbook 执行脚本](#例子一使用-curveadm-playbook-执行脚本)
+  - [例子二：使用 curveadm palybook 执行带参数的脚本](#例子二使用-curveadm-palybook-执行带参数的脚本)
```

Comment:
Do you need to consider multiple parameters？

Reply:
> Do you need to consider multiple parameters？

support